platform/saivpp/vpplib/SwitchStateBaseAcl.cpp

```diff
@@ -1290,6 +1639,11 @@ sai_status_t SwitchStateBase::aclBindUnbindPorts(
 			is_bind ? "bound to": "unbound from", hwif_name.c_str());
     }
 
+    for(auto hwif: hwif_list) {
```

Comment:
Does this correctly update the ports_map in the unbind case?

Reply:
Move the ports_map unbind update above directly in the unbind case